adafruit_is31fl3731/__init__.py

```diff
@@ -275,13 +295,13 @@ def blink(self, rate=None):
         self._register(_CONFIG_BANK, _BLINK_REGISTER, rate & 0x07 | 0x08)
         return None
 
-    def fill(self, color=None, blink=None, frame=None):
+    def fill(self, color: int = None, blink: bool = False, frame: int = 0) -> int:
```

Comment:
color can be `Optional[int]`

It looks like there is some logic below that looks for and handles `fram=None` value specifically so I think we should change it's default value back to `None` instead of `0`. That would also make it's type into `Optional[int]`